### DISCUSSION TITLE
How to change selected listboxitem style ?

### DISCUSSION THREAD

**Discussion by jim-jiang-github** [AvaloniaUI/Avalonia#11920] [Q&A] [answered]
![image](https://github.com/AvaloniaUI/Avalonia/assets/29189410/81d858cc-c112-44ba-b134-ac96c22ccddb)

I do this but got unexpected result.

**Comment by rabbitism** [comment, marked as answer]
Please check the fluent theme source code. 
https://github.com/AvaloniaUI/Avalonia/blob/11.0.0-preview4/src/Avalonia.Themes.Fluent/Controls/ListBoxItem.xaml

**Reply by timunie** [reply]
HINT: The link you posted is for 11.0 preview4. I'd suggest to switch branch to master if you use 11.0 RC1 or to stable/0.10.x if you use current stable version
